src/test/local-file-system.test.ts

```diff
@@ -8,25 +8,88 @@ describe('LocalFileSystem', () => {
   let tempDir: string;
 
   beforeEach(async () => {
-    tempDir = await fs.mkdtemp(path.join(tmpdir(), 'reporter-test-'));
+    tempDir = await fs.mkdtemp(path.join(tmpdir(), 'lfs-'));
+    await fs.writeFile(
+      path.join(tempDir, 'package.json'),
+      JSON.stringify({name: 'pkg', version: '1.0.0'})
+    );
   });
 
   afterEach(async () => {
     await fs.rm(tempDir, {recursive: true, force: true});
   });
 
-  describe('fileExists', () => {
-    it('should return false when tsconfig.json does not exist', async () => {
-      const fileSystem = new LocalFileSystem(tempDir);
-      const hasConfig = await fileSystem.fileExists('/tsconfig.json');
-      expect(hasConfig).toBe(false);
-    });
-
-    it('should return true when file exists', async () => {
-      await fs.writeFile(path.join(tempDir, 'tsconfig.json'), '{}');
-      const fileSystem = new LocalFileSystem(tempDir);
-      const hasConfig = await fileSystem.fileExists('/tsconfig.json');
-      expect(hasConfig).toBe(true);
-    });
+  it('should report false for a missing file and true for an existing file', async () => {
```

Comment:
this should probably be two tests

Reply:
looking at it, this and the `readFile` tests were probably organised better before, no?

roughly:

- `fileExists`
  - test that it is true for files that exist
  - test that it is false for files that dont exist
- `readFile`
  - test that it returns files that exist
  - test that it throws for files that don't exist